rust/core-lib/src/view.rs

```diff
@@ -113,6 +116,18 @@ pub struct Replace {
     pub preserve_case: bool
 }
 
+// todo: also used in tab.rs
+#[derive(Debug, Default)]
+struct Counter(Cell<usize>);
```

Comment:
Assuming this is duplicated and not moved, I would just use reuse the existing type (making it pub(crate))?